listenbrainz/webserver/templates/emails/year_in_music.html

```diff
@@ -1,71 +1,55 @@
+
+    <title>Your Year in Music report</title>
+    <meta property="og:title" content="Your Year in Music report">
+
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <!--[if gte mso 9]>
+    <xml>
+    <o:OfficeDocumentSettings>
+    <o:AllowPNG/>
+    <o:PixelsPerInch>96</o:PixelsPerInch>
+    </o:OfficeDocumentSettings>
+    </xml>
+    <![endif]-->
+  </head>
+  <body style="padding:0px; margin:0px">
+    <main style="max-width: 600px;margin-left: auto;margin-right: auto;border-color: transparent;background-color: #ffffff;font-family: 'Roboto', sans-serif;font-weight: 300">
+      <div style="padding: 2em;border-style: none;background-color: transparent;margin: auto;text-align: center;">
+        <img alt="Year in music 2022 logo" src="cid:{{ logo_cid }}">
+      </div>
+      <header style="width: 100%;height: auto;padding: 2em 1em;background-color: #ff0e25; color:#ffffff;text-align: center;">
+        <p>
+          <span style="text-transform: capitalize;">{{user_name}}</span>, your ListenBrainz Year in Music 2022 is now available.
+        </p>
+        <a href="https://listenbrainz.org/user/{{user_name}}/year-in-music/" style="font-weight: 700;margin-top:1em;color:#ffffff">Click here</a>
+      </header>
+      <div style="padding:1em;color: rgba(30, 30, 30, 1);width: auto;height: auto;font-size: 12px;text-align: center;line-height: normal;">
+        <p>
+          We have compiled the Year in Music report for 2022, which summarises of your listening habits throughout the year.
+          Some of the statistics you can expect to see include your most active listening day of the week, your listening days throughout the year, and users who listened to similar music to you. Additionally, we have included information about the countries where your favorite artists are from.
+        </p>
+        <p>
+        To top it off, we have also generated two new playlists for you to discover even more music in 2023.
+        We hope you enjoy the report and the new playlists.
+        <p>
+        <p>
+        Happy listening, and thanks for being part of ListenBrainz!
         <br>
-        The ListenBrainz team: akshaaatt, alastairp, lucifer, mayhem and monkey.
-    </p>
-    <p>
-        P.S. Heads up! This page loads many album covers; if you are on a mobile device you might want to wait
-        until you are connected to wifi.
-    </p>
-</div>
+        Wishing you a wonderful 2023, from the ListenBrainz team.
+        </p>
+      </div>
+      <div style="padding:0.5em;background-color: #f2f2f2;color: rgba(30, 30, 30, 1);font-size: 9px;text-align: center;font-weight: 300">
+            Whatever small print we usually have at the bottom of our emails, links and junk
```

Comment:
Here this text is only a placeholder. Do we have links and stuff we want to put in there? (go to account/read in browser/opt-out from email, etc)
If so let's put those in. If not let's remove the whole div

Reply:
We don't really have an opt-out at the moment, but I thinks to LB, MB and MeB could be appropriate here, to give the user links to the projects that contributed to this report. But this is not a strong feeling, if we agree to nuke it, then lets do that.

Reply:
Nuked. We probably still have a few hours to add it back if we want to.